Cannot compile test code

```
$ ./gradlew test
:compileJava UP-TO-DATE
:processResources UP-TO-DATE
:classes UP-TO-DATE
:compileTestJava
/Users/ingmar/workspace/speech-analysis-webapp/src/test/java/com/speechanalysis/selenium/LoginTest.java:13: error: cannot find symbol
        WebDriver driver = new HtmlUnitDriver();
                               ^
  symbol:   class HtmlUnitDriver
  location: class LoginTest
/Users/ingmar/workspace/speech-analysis-webapp/src/test/java/com/speechanalysis/selenium/LoginTest.java:15: error: cannot find symbol
        WebElement element = driver.findElement(By.id("username"));
        ^
  symbol:   class WebElement
  location: class LoginTest
/Users/ingmar/workspace/speech-analysis-webapp/src/test/java/com/speechanalysis/selenium/LoginTest.java:15: error: cannot find symbol
        WebElement element = driver.findElement(By.id("username"));
                                                ^
  symbol:   variable By
  location: class LoginTest
3 errors
:compileTestJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 1.633 secs
```

--- a/build.gradle
+++ b/build.gradle
@@ -32,8 +32,7 @@ dependencies {
 	compile('org.springframework.boot:spring-boot-starter-web')
 	compile 'org.webjars:jquery:3.0.0'
 	compile('com.h2database:h2')
-	//runtime('org.postgresql:postgresql:9.4.1208.jre7')
-	compile('org.seleniumhq.selenium:selenium-java:3.0.0-beta2')
+	compile('org.seleniumhq.selenium:selenium-java:2.9.0')
 	testCompile('org.springframework.boot:spring-boot-starter-test') 
 }
 

--- a/src/main/resources/application.properties.version
+++ b/src/main/resources/application.properties.version
@@ -7,3 +7,4 @@
 #spring.datasource.username=
 #spring.datasource.password=
 #server.port=8080
+spring.messages.basename=validation

--- a/src/test/java/com/speechanalysis/selenium/LoginTest.java
+++ b/src/test/java/com/speechanalysis/selenium/LoginTest.java
@@ -1,7 +1,10 @@
 package com.speechanalysis.selenium;
 
 import org.junit.Test;
+import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.htmlunit.HtmlUnitDriver;
 
 /**
  * Created by plam on 25/08/16.

